lib/proto.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+const { EventEmitter } = require('events')
+const SonicBoom = require('sonic-boom')
+const flatstr = require('flatstr')
+const {
+  lsCacheSym,
+  getLevelValSym,
+  setLevelValSym,
+  setLevelSym,
+  getLevelSym,
+  chindingsSym,
+  asJsonSym,
+  writeSym,
+  needsMetadataGsym
+} = require('./symbols')
+const {
+  lsCache,
+  setLevelVal,
+  getLevelVal,
+  getLevel,
+  setLevel,
+  addLevel,
+  setLevelState,
+  isLevelEnabled,
+  isStandardLevelVal
+} = require('./levels')
+const {
+  noop,
+  asChindings,
+  asJson
+} = require('./tools')
+const {
+  version,
+  LOG_VERSION
+} = require('./meta')
+// note: use of class is satirical
+const constructor = class Pino {}
```

Comment:
this is the best!

Reply:
What is the actual justification for it?

Reply:
@jsumners – debugging/profiling purposes. Otherwise it's "EventEmitter". It's just a (next to zero cost) nicety to have the object labelled as "Pino"

Reply:
How about `prototype[Symbol.toStringTag] = 'Pino'`?

Reply:
![image](https://user-images.githubusercontent.com/1190716/41296021-39bb5722-6e5c-11e8-95ed-c2c8ecee1762.png)

Reply:
:-/